src/main/java/school/hei/haapi/endpoint/rest/controller/StudentController.java

```diff
@@ -41,19 +44,18 @@ public Student getStudentById(
   @GetMapping("/students")
   public List<Student> getStudents(
       @RequestParam PageFromOne page, @RequestParam("page_size") BoundedPageSize pageSize) {
-    return userService
-        .getByRole(User.Role.STUDENT, page, pageSize).stream()
+    return userService.getByRole(User.Role.STUDENT, page, pageSize).stream()
         .map(userMapper::toRestStudent)
         .collect(toUnmodifiableList());
   }
 
   @PutMapping("/students")
-  public List<Student> saveAll(@RequestBody List<Student> toSave) {
+  public List<Student> saveAll(@RequestBody List<Student> toWrite) {
+    if (toWrite.size() > EventConf.MAX_EVENT_REQUEST_ENTRY) {
+      throw new BadRequestException("students to create or update must be ≤ 10");
```

Comment:
Our exception message starts with a capital letter. Also, use MAX_EVENT_REQUEST_ENTRY instead of the magic number 10